From e52c4d30415d57822f39a9c65c8d4c51dded185c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lawrence=20Vel=C3=A1zquez?= <larryv@macports.org>
Date: Wed, 4 Feb 2015 14:54:57 -0500
Subject: [PATCH 06/16] (MP) BuildSystem: Add targets for MacGui and CLI

This allows the GUI and CLI to be built and installed independently.
---
 macosx/module.xcodebuild | 16 +++++++++++++++-
 1 file changed, 15 insertions(+), 1 deletion(-)

diff --git a/macosx/module.xcodebuild b/macosx/module.xcodebuild
index 26b34a9..e16a515 100644
--- a/macosx/module.xcodebuild
+++ b/macosx/module.xcodebuild
@@ -16,9 +16,23 @@ clean: macosx.clean
 install: macosx.install
 xclean: clean
 
+.PHONY: macosx.gui.build macosx.gui.install macosx.cli.build macosx.cli.install
+
+macosx.gui.build:
+	$(call MACOSX.XCODE,HandBrake,build)
+
+macosx.gui.install:
+	$(call MACOSX.XCODE,HandBrake,install)
+
+macosx.cli.build:
+	$(call MACOSX.XCODE,HandBrakeCLI,build)
+
+macosx.cli.install:
+	$(call MACOSX.XCODE,HandBrakeCLI,install)
+
 ###############################################################################
 
-MACOSX.goals = $(filter-out build clean install,$(MAKECMDGOALS))
+MACOSX.goals = $(filter-out build clean install macosx.gui.build macosx.gui.install macosx.cli.build macosx.cli.install,$(MAKECMDGOALS))
 
 $(MACOSX.goals): __goals__
 	@true
-- 
2.3.0

